theano/tensor/elemwise.py

```diff
@@ -492,6 +493,8 @@ class Elemwise(OpenMPOp):
 
     """
 
+    __props__ = ("scalar_op", "inplace_pattern")
```

Comment:
@abergeron @lamblin should the openmp parameter be in this list? I'm not sure of that. It isn't used in \__hash__ and \__eq__. Can you confirm it is fine to not put it here? This would be equivalent to what we do right now.